puppet/modules/jenkins_master/manifests/init.pp

```diff
@@ -0,0 +1,201 @@
+    install_java => false,
+    lts => true,
+    config_hash => {
+      'JENKINS_JAVA_OPTIONS' => {
+        'value' => '-Djava.awt.headless=true -Djenkins.install.runSetupWizard=false -Xms2048m -Xmx2048m',
+      }
+    }
+  }
+
+  $plugins = {
+    'ace-editor'                         => '1.1',
+    'analysis-core'                      => '1.95',
+    'ansible'                            => '1.0',
+    'ansicolor'                          => '0.5.2',
+    'ant'                                => '1.8',
+    'antisamy-markup-formatter'          => '1.5',
+    'apache-httpcomponents-client-4-api' => '4.5.5-2.1',
+    'authentication-tokens'              => '1.3',
+    'blueocean'                          => '1.5.0',
+    'blueocean-autofavorite'             => '1.2.2',
+    'blueocean-bitbucket-pipeline'       => '1.5.0',
+    'blueocean-commons'                  => '1.5.0',
+    'blueocean-config'                   => '1.5.0',
+    'blueocean-core-js'                  => '1.5.0',
+    'blueocean-dashboard'                => '1.5.0',
+    'blueocean-display-url'              => '2.2.0',
+    'blueocean-events'                   => '1.5.0',
+    'blueocean-git-pipeline'             => '1.5.0',
+    'blueocean-github-pipeline'          => '1.5.0',
+    'blueocean-i18n'                     => '1.5.0',
+    'blueocean-jira'                     => '1.5.0',
+    'blueocean-jwt'                      => '1.5.0',
+    'blueocean-personalization'          => '1.5.0',
+    'blueocean-pipeline-api-impl'        => '1.5.0',
+    'blueocean-pipeline-editor'          => '1.5.0',
+    'blueocean-pipeline-scm-api'         => '1.5.0',
+    'blueocean-rest'                     => '1.5.0',
+    'blueocean-rest-impl'                => '1.5.0',
+    'blueocean-web'                      => '1.5.0',
+    'bouncycastle-api'                   => '2.16.2',
+    'brakeman'                           => '0.12',
+    'branch-api'                         => '2.0.20',
+    'build-flow-plugin'                  => '0.20',
+    'build-keeper-plugin'                => '1.3',
+    'build-metrics'                      => '1.3',
+    'build-pipeline-plugin'              => '1.5.8',
+    'build-timeout'                      => '1.19',
+    'buildgraph-view'                    => '1.8',
+    'bulk-builder'                       => '1.5',
+    'checkstyle'                         => '3.50',
+    'cloudbees-bitbucket-branch-source'  => '2.2.11',
+    'cloudbees-folder'                   => '6.4',
+    'cobertura'                          => '1.12.1',
+    'command-launcher'                   => '1.2',
+    'conditional-buildstep'              => '1.3.6',
+    'config-file-provider'               => '2.18',
+    'configurationslicing'               => '1.47',
+    'copy-to-slave'                      => '1.4.4',
+    'copyartifact'                       => '1.39.1',
+    'create-fingerprint'                 => '1.2',
+    'credentials'                        => '2.1.16',
```

Comment:
This plugin is included by default in the jenkins module and causes a conflict so we can omit it here or comment it out with a comment. Take a look at:

https://github.com/voxpupuli/puppet-jenkins/blob/master/manifests/config.pp#L9
https://github.com/voxpupuli/puppet-jenkins/blob/ef41da82f100a10f2dcfd04c6286d504452d3988/manifests/params.pp#L29-L31 (note you can't look at this in master cause we don't use master of this module)

Reply:
updated